### PR TITLE
Inlanguage seo enhancement

### LINKED ISSUE #7634: SEO enhancement: Add inLanguage object to schema.org
**Is your feature request related to a problem? Please describe.**
A new `inLanguage` property is available in Schema.org - we should add this to describe our pages.

**Describe the solution you'd like**
- On all page types, add an inLanguage property to describe the language of the page.
- We need a new field in the service config for the language name https://github.com/bbc/simorgh/blob/latest/src/app/lib/config/services/afaanoromoo.js#L9
- We should use the `lang` field within each service config as language code.
- The `inLanguage` property should be added to this file, for all page types https://github.com/bbc/simorgh/blob/latest/src/app/containers/LinkedData/index.jsx#L10

Example:
```
inLanguage {
    "@type": "Language",
    "name": "Spanish",
    "alternateName": "es"
}
```

The language codes used for `alternateName` may not be the same ones we use elsewhere as they follow a different spec.  We should therefore use:

Service/Variant | Language name | Language code
--- | --- | ---
afaanoromoo | Oromo | om
afrique | French | fr
amharic | Amharic | am
arabic | Arabic | ar
azeri | Azerbaijani | az
bengali | Bengali | bn
burmese | Burmese | my
gahuza | Kinyarwanda | rw
gujarati | Gujarati | gu
hausa | Hausa | ha
hindi | Hindi | hi
igbo | Igbo | ig
indonesia | Indonesian | id
japanese | Japanese | ja
korean | Korean | ko
kyrgyz | Kyrgyz | ky
marathi | Marathi | mr
mundo | Spanish | es
nepali | Nepali | ne
pashto | Pashto | ps
persian | Persian  | fa
pidgin | Nigerian Pidgin | pcm
portuguese | Portuguese | pt-BR
punjabi | Punjabi | pa
russian | Russian | ru
serbian latin | Serbian | sr-Latn
serbian cyrillic | Serbian | sr-Cyrl
sinhala | Sinhala | si
somali | Somali | so
swahili | Swahili | sw
tamil | Tamil | ta
telugu | Telugu | te
thai | Thai | th
tigrinya | Tigrinya | ti
turkce | Turkish | tr
ukchina simp | Chinese | zh-Hans
ukchina trad | Chinese | zh-Hant
ukrainian | Ukrainian | uk
ukrainian (russian language) | Russian | ru-UA
urdu | Urdu | ur
uzbek | Uzbek | uz-Cyrl
vietnamese | Vietnamese | vi
yoruba | Yoruba | yo
zhongwen simp | Chinese | zh-Hans
zhongwen trad | Chinese | zh-Hant
news | English | en
archive | English | en
scotland | English | en

**Describe alternatives you've considered**
N/A

**Testing notes**
- Check across all page types

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**
https://schema.org/inLanguage

### PR DESCRIPTION
**Pull request by rhenshaw56**
Resolves #7634 

**Overall change:**
_Adds new inLanguage property to Schema.org metadata._

**Code changes:**

- _adds new `languageName` field to all service configs._
- _added inLanguage object to shema.org metadata._

---

- [X] I have assigned myself to this PR and the corresponding issues
- [X] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [X] I have assigned this PR to the Simorgh project

**Testing:**

- [X] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [X] This PR requires manual testing

### CODE REVIEW

File: src/app/containers/LinkedData/index.jsx [hunk 1]
```diff
@@ -66,6 +69,12 @@ const LinkedData = ({
     name: seoTitle,
   };
 
+  const inLangauge = {
```
Comment (karinathomasbbc) [inline src/app/containers/LinkedData/index.jsx]: Typo - should be `inLanguage`

Comment (rhenshaw56) [reply src/app/containers/LinkedData/index.jsx]: 😱

File: src/app/lib/config/services/cymrufyw.js [hunk 2]
```diff
@@ -39,6 +39,7 @@ export const service = {
     datetimeLocale: `cy`,
     service: 'cymrufyw',
     serviceName: 'Cymru Fyw',
+    languageName: '',
```
Comment (karinathomasbbc) [inline src/app/lib/config/services/cymrufyw.js]: Is the default 'English' if this value is blank?

Comment (rhenshaw56) [reply src/app/lib/config/services/cymrufyw.js]: not sure. It wasn't in the list. any idea @jimjohnsonrollings ?

Comment (jimjohnsonrollings) [reply src/app/lib/config/services/cymrufyw.js]: We don't serve any Cymru Fyw pages, but happy for it to have the languageName = `Welsh` if we need one

Comment (jimjohnsonrollings) [reply src/app/lib/config/services/cymrufyw.js]: If we need a value for Naidheachdan too, we should use `Scottish Gaelic`

Comment (rhenshaw56) [reply src/app/lib/config/services/cymrufyw.js]: Thanks Jim.

**Review by karinathomasbbc — APPROVED** [review]

**Review by chris-hinds — APPROVED** [review]

**Comment by paruchurisilpa** [comment]
* Go to: https://search.google.com/structured-data/testing-tool/u/0/
* Checked few languages and they all are using correct inLanguage

For eg;
     --- 
inLanguage |  
-- | --
@type | Language
name | Spanish
alternateName | es

LGTM..

**Pull request merged**